alerts/storage_alerts.libsonnet

```diff
@@ -43,7 +43,7 @@
               and
               predict_linear(kubelet_volume_stats_available_bytes{%(prefixedNamespaceSelector)s%(kubeletSelector)s}[%(volumeFullPredictionSampleTime)s], 4 * 24 * 3600) < 0
             ||| % $._config,
-            'for': '5m',
+            'for': '1h',
             labels: {
               severity: 'critical',
```

Comment:
not blocking this, but if we have four days to react to this, then critical is probably not appropriate

Reply:
Well, depending on the rate of increase, it can be critical. The alerts could probably be defined better, but for 4 days notice, I think the current fix is an improvement on the reliability of the 4 day alert.

What we used to do in Zabbix, is have 2 predictive triggers: one high severity, for rapid change “Fix it now!" and one lower severity “Fix it in the next morning" for a slower trend.

Reply:
Yes that makes sense we did the same for [host disk alerting](https://github.com/prometheus/node_exporter/blob/f9d2bbe85476395c764188a445997960e4dae718/docs/node-mixin/alerts/alerts.libsonnet#L28), I would say we do the exact same scheme.

Reply:
@brancz I like consistency. Do you propose defining the alerts for PVCs the same way they're currently defined for node-exporter?
I imagine, some people may have an issue with 24h notice, since it doesn't cover the weekend. (:
I wouldn't mind picking this up if there was an issue about it. I'd rather not solve problems that don't need solving.

Reply:
There are two alerts in the node-mixin. Yes I think if we leave the severity configurable like in the node mixin we should be ok.